CotEditor/Localizables/InAppPurchase.xcstrings

```diff
@@ -76,6 +76,12 @@
             "value" : "Uma saborosa xícara"
           }
         },
+        "ru" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Разовая поддержка"
```

Comment:
A more direct translation to the base label ("A tasty cup") would be preferable.